composer.json

```diff
@@ -15,7 +15,8 @@
         "ezsystems/comments-bundle": "@dev",
         "ezsystems/platform-ui-assets-bundle": "~0.1",
         "whiteoctober/breadcrumbs-bundle": "~1.0.1",
-        "knplabs/knp-menu-bundle": "1.*"
+        "knplabs/knp-menu-bundle": "1.*",
+        "components/bootstrap": "3.3.2"
```

Comment:
Ok, up to you :)
I didn't see anything risky in [3.3.0 to 3.3.2](https://github.com/twbs/bootstrap/compare/v3.3.0...v3.3.2#diff-9476f0f857cc69f29e7cdb7865d409e2) update, and this being a demo bundle I thought we could threat it like we do with php packages, but lets avoid breaks.